nipype/utils/filemanip.py

```diff
@@ -673,6 +674,65 @@ def dist_is_editable(dist):
     return False
 
 
+def makedirs(path, exist_ok=False):
```

Comment:
I wonder weather using the same name as the `os` can be confusing.

Reply:
The idea is to be able to remove this safely when we only support Python 3